cmd/tke-installer/app/installer/installer.go

```diff
@@ -1370,6 +1370,10 @@ func (t *TKE) prepareBaremetalProviderConfig(ctx context.Context) error {
 			Name: "provider-config",
 			File: baremetalconstants.ConfDir + "*.yaml",
 		},
+		{
+			Name: "provider-config",
```

Comment:
the config map name conflict with previous one

Reply:
should write into the same ConfigMap.
`CreateOrUpdateConfigMapFromFile` will update the ConfigMap.